server/ipqueue_test.go

```diff
@@ -42,7 +42,7 @@ func TestIPQueueBasic(t *testing.T) {
 	}
 
 	// Try to change the max recycle size
-	q2 := newIPQueue[int](s, "test2", ipQueue_MaxRecycleSize(10))
+	q2 := newIPQueue[int](s, "test2", ipQueue_MaxRecycleSize[int](10))
```

Comment:
I like how this works, but the name of the options might need tweaking.

ipqMaxRecycleSize() - nix the long name and the '_' I think here.